src/Resolvers/Microsoft.DotNet.NativeWrapper/Interop.cs

```diff
@@ -81,9 +109,21 @@ internal static class Windows
             internal static extern int hostfxr_get_available_sdks(
                 string exe_dir,
                 hostfxr_get_available_sdks_result_fn result);
+
+            [UnmanagedFunctionPointer(CallingConvention.Cdecl, CharSet = UTF16)]
```

Comment:
Why not CharSet = Auto and have just one version instead of Unix/Windows split?

Reply:
Good point, just made that change.